modules/generate-diagnostic-bundle-using-roxctl-cli.adoc

```diff
@@ -1,12 +1,54 @@
 // Module included in the following assemblies:
 //
 // * configuration/generate-diagnostic-bundle.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="generate-diagnostic-bundle-using-roxctl-cli_{context}"]
 = Generating a diagnostic bundle by using the roxctl CLI
 
 You can generate a diagnostic bundle by using the `roxctl` CLI.
 
+.Prerequisites
+* To generate a diagnostic bundle, you need `read` permission for the `Administration` resource. This is required for versions of the `DebugLogs` resource older than version 3.73.0.
+
+* You have configured the `ROX_PASSWORD` and `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Run the following command to set the `ROX_PASSWORD` and `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_PASSWORD=_<rox_password>_ && export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_ <1>
+----
++
+<1> For `<rox_password>`, specify the {rh-rhacs-first} administrator password.
+
+* Alternatively, you can configure the `ROX_API_TOKEN` and `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Run the following command to set the `ROX_API_TOKEN` and `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_API_TOKEN=_<api_token>_ && export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_
+----
+
+.Procedure
+
+* Run the following command to generate a diagnostic bundle:
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" -p "$ROX_PASSWORD" central debug download-diagnostics
+----
+
+* Alternatively, you can generate a diagnostic bundle using the following command:
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug download-diagnostics
```

Comment:
**thought:** I think they can run this if they have already set an API token. So maybe take the steps from prerequisites and put them here.